### PR TITLE
Fixed some invalid HTML/CSS

### PR DESCRIPTION
**Pull request by skoehler47**
This pull requests contains some fixes for invalid HTML and CSS, which is automatically created by Wt - as described in [Issue 10482](https://redmine.webtoolkit.eu/issues/10482)

### CODE REVIEW

**Review by emweb — CHANGES_REQUESTED** [review]
Thanks for your contribution. I have one small remark.

We indeed added the self-closing tag fixing to `<head-matter>` but didn't take special care of `link` and `meta` tags.

`cursor: hand;` is a leftover cross browser compatibility thing. It's fine to remove it now.

Roel

File: src/web/DomElement.C [hunk 1]
```diff
@@ -697,7 +697,7 @@ std::string DomElement::cssStyle() const
     else if ((p >= static_cast<unsigned int>(Property::StylePosition)) &&
              (p < static_cast<unsigned int>(Property::LastPlusOne))) {
       if ((j->first == Property::StyleCursor) && (j->second == "pointer")) {
-        style << "cursor:pointer;cursor:hand;";
+        style << "cursor:pointer;";
```
[the remark this answers is not in the document]
Comment (skoehler47) [reply src/web/DomElement.C]: I removed that branch and updated the commit.

[the remark this answers is not in the document]
Comment (emweb) [reply src/web/DomElement.C]: Thanks!

[the remark this answers is not in the document]
Comment (skoehler47) [reply src/web/DomElement.C]: Just looked it up. Didn't know old Internet Explorer actually used 'cursor: hand'... Thanks for the feedback :)

**Pull request merged**